modules/rbac-resource-definitions.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: CONCEPT
+[id="resource-definitions_{context}"]
+= Resource definitions
+
+[role="_abstract"]
+
+{product-title} includes multiple resources.
+The following table lists the resources and describes the actions that users can perform with the `read` or `write` permission.
+
+[cols="1,2,2"]
+|===
+| Resource | Read permission | Write permission
+
+| APIToken
+| List existing API tokens.
+| Create new API tokens or revoke existing tokens.
+//TODO: Add link to API tokens
+
+| Alert
+| View existing policy violations.
+| Resolve or edit policy violations.
+//TODO: Add link to policy violations
+
+| AllComments
+| N/A
+| Delete comments from other users.
+All users can edit and delete their own comments by default.
+To add and remove comments or tags, you need a role with `write` permission for the resource you are modifying.
+//TODO: Add link to Comments and tags
+
+| AuthPlugin
+| View existing Authentication Plugins
```

Comment:
I can't remember if it's come up or not yet, but we do use "plug-in" with a hyphen. But it's fine to keep no hyphen if that's how it's used regularly throughout these docs.

However, also does this need to be capitalized?